doc/classes/AudioEffectReverb.xml

```diff
@@ -124,20 +127,28 @@
 	</methods>
 	<members>
 		<member name="damping" type="float" setter="set_damping" getter="get_damping">
+			Widens or narrows the stereo image of the reverb tail. 1 means very large. Value can range from 0 to 1. Default value: [code]1[/code].
```

Comment:
This isn't clear. 1 means "very large" what? How does that relate to widening or narrowing the tail.

Reply:
DO you think "1 means fully widens" is better ?